test/integ/zeroDimBuffer/src/zeroDimBuffer.cpp

```diff
@@ -0,0 +1,137 @@
+/* Copyright 2021 Andrea Bocci
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+// make sure the CPU_B_SEQ_T_SEQ backend is always available
+#ifndef ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED
+#    define ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED
+#endif // ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED
+#include <alpaka/alpaka.hpp>
+#include <alpaka/test/acc/TestAccs.hpp>
+
+#include <catch2/catch.hpp>
+
+#include <cstdint>
+#include <iostream>
+
+constexpr int memset_value(int c)
+{
+    int v = c & 0xFF;
+    return (v | v << 8 | v << 16 | v << 24);
+}
```

Comment:
This code assumes that `int` is 32bit. Can you please add a `static_assert` for that? Or write the code generically.

Reply:
`int` is 32 bits on any single architecture I have access to.

Reply:
Changed to support any arbitrary trivial type.